### DISCUSSION TITLE
Where is this text/link coming from?

### DISCUSSION THREAD

**Discussion by rastiazul** [community/community#23009] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, so this is my fourth atempt at creating a site and the only error I am getting is some long text showing on the bottom of every page on my site. Check it out here:</p>
<p><a href="https://rastiazul.github.io//blogs/" rel="nofollow noopener">https://rastiazul.github.io//blogs/</a></p>
<p>It links to %7B, which doesnt exist. </p>
<p>This is part of it, if you dont feel like clicking on it:</p>
<pre><code>- { "api_url": "https://api.github.com", "baseurl": "", "build_revision": "e60c7a84c5d089a5b2105d1aab193302ebffd228", "clone_url": "https://github.com/rastiazul/rastiazul.github.io.git", "contributors": [ { "login": "tmjam", "id": 964223, "node_id": "MDQ6VXNlcjk2NDIyMw==", "avatar_url": "https://avatars3.githubusercontent.com/u/964223?v=4", "gravatar_id": "", "url": "https://api.github.com/users/tmjam", "html_url": "https://github.com/tmjam", "followers_url": "https://api.github.com/users/tmjam/followers", "following_url": "https://api.github.com/users/tmjam/following{/other_user}", "gists_url": "https://api.github.com/users/tmjam/gists{/gist_id}", "starred_url": "https://api.github.com/users/tmjam/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/tmjam/subscriptions", "organizations_url": "https://api.github.com/users/tmjam/orgs", "repos_url":
</code></pre>
<p>Thank you!</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/rastiazul">@rastiazul</a>,</p>
<p>The site itself seems to be working fine currently. Are you still not able to update it?</p>
<p>Thanks! </p>

**Comment by rastiazul** [comment, marked as answer]
<p>It is fine now, thank you. There was some weird configuration in the template that messed up the site if I removed the github links</p>
